src/MahApps.Metro/MahApps.Metro.Shared/Controls/Helper/WatermarkAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace MahApps.Metro.Controls.Helper
+{
+#if NET4_5
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+    public class WatermarkAttribute : Attribute
+    {
+        public WatermarkAttribute(string caption)
+        {
+            this.Caption = caption;
+        }
+
+        public string Caption { get; set; }
+    }
+#endif
+}
```

Comment:
Why is it necessary to introduce a new attribute? Is the Display attribute too bad?